pages/common/pio-ci.md

```diff
@@ -0,0 +1,29 @@
+# pio ci
+
+> Build PlatformIO projects with an arbitrary source code structure.
+> This will create a new temporary project which the source code will be copied into.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/cmd_ci.html>.
+
+- Build a PlatformIO project in the default system temporary directory and delete it afterwards:
+
+`pio ci {{path/to/project}}`
+
+- Build a PlatformIO project and specify specific libraries:
+
+`pio ci --lib {{path/to/library_directory}} {{path/to/project}}`
+
+- Build a PlatformIO project and specify a specific board (`pio boards` lists all of them):
+
+`pio ci --board {{board}} {{path/to/project}}`
+
+- Build a PlatformIO project in a specific directory:
+
+`pio ci --build-dir {{path/to/build_directory}} {{path/to/project}}`
+
+- Build a PlatformIO project and do not delete the build directory:
```

Comment:
```suggestion
- Build a PlatformIO project and don't delete the build directory:
```

Reply:
I think that's just a matter of taste. I decided for `do not`, because it's more formal and puts the focus on it, since it's the important part of the description.

Reply:
Sure. 
I _do not_ mind, I just feel it rolls off the tongue a little nasty compared to don't

Reply:
Maybe @sbrl has an opinion?

Reply:
Re-reading it a few days later, I actually prefer your suggestion